mongodbatlas/data_source_mongodbatlas_maintenance_window_test.go

```diff
@@ -34,21 +34,19 @@ func TestAccDataSourceMongoDBAtlasMaintenanceWindow_basic(t *testing.T) {
 }
 
 func TestAccDataSourceMongoDBAtlasMaintenanceWindow_basicWithStartASAP(t *testing.T) {
-	var maintenance matlas.MaintenanceWindow
-
-	projectID := os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+	t.Skip()
 
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:  func() { testAccPreCheck(t) },
 		Providers: testAccProviders,
-		Steps: []resource.TestStep{
-			{
+		Steps:     []resource.TestStep{
+			/*{
```

Comment:
Why this so much in this test commented out?

Reply:
because I skip, if you want I can delete the code related with ASAP?

Reply:
If we don't need it then removing it is fine - much cleaner than leaving unused code.